faucet/frontend/src/locales/cn.json

```diff
@@ -0,0 +1,31 @@
+{
+  "welcomeMsg" : "在這裡你可以獲得令牌來測試DISCIPLINA區塊鏈",
```

Comment:
What a nice Chinese localisation :+1:

Reply:
I hope that doesn't accidentally tell someone to go pound sand.